### PR TITLE
Update bridges subtree

### PR DESCRIPTION
**Pull request by serban300**
- Backport the latest changes from the bridges repo: https://github.com/paritytech/parity-bridges-common

- moved the bridges subtree from `./cumulus` to `./`

### CODE REVIEW

**Comment by acatangiu** [comment]
We were actually talking that “bridges” dir could be moved from under cumulus directly into root: “cumulus/bridges“ -> “bridges”.
When we’ll migrate parity-bridges-common repo to monorepo “bridges” dir, it’ll contain other components too not just bridge-hub pallets and primitives.

@bkchr @ggwpez wdyt?

**Comment by bkchr** [comment]
Sounds like a good idea to also move the bridges stuff over! Having it on the top level also sounds about right to me!

**Review by svyatonik — APPROVED** [review]
Shall we also change the [bridges subtree rule](https://github.com/paritytech/polkadot-sdk/blob/master/.github/pr-custom-review.yml#L39-L45) here?

**Comment by serban300** [comment]
> Sounds like a good idea to also move the bridges stuff over! Having it on the top level also sounds about right to me!

Ok, I will adjust the PR then and move the subtree under root.

> Shall we also change the [bridges subtree rule](https://github.com/paritytech/polkadot-sdk/blob/master/.github/pr-custom-review.yml#L39-L45) here?

Good point. Will do.

**Comment by serban300** [comment]
Updated the PR in order to move the bridges subtree under root. Sorry, I had to force push.

@svyatonik @acatangiu @bkchr PTAL when you have time.

**Review by lexnv — APPROVED** [review]

File: bridges/scripts/verify-pallets-build.sh [hunk 1]
```diff
@@ -89,17 +89,26 @@ rm -rf $BRIDGES_FOLDER/scripts/update-weights-setup.sh
...
 rm -f $BRIDGES_FOLDER/Dockerfile
+rm -f $BRIDGES_FOLDER/rustfmt.toml
 
 # let's fix Cargo.toml a bit (it'll be helpful if we are in the bridges repo)
 if [[ ! -f "Cargo.toml" ]]; then
 	cat > Cargo.toml <<-CARGO_TOML
+	[workspace.package]
+	authors = ["Parity Technologies <admin@parity.io>"]
+	edition = "2021"
+	repository = "https://github.com/paritytech/parity-bridges-common.git"
```
[the remark this answers is not in the document]
Comment (serban300) [reply bridges/scripts/verify-pallets-build.sh:109]: I think we'll delete this script when we migrate the full repo. Doesn't look like we'll need it anymore. And our crates will inherit the `[workspace.package]` from `polkadot-sdk`

**Comment by serban300** [comment]
> Looks good!
> 
> One more thing we should do is rename `bridges/modules/` -> `bridges/pallets/`. We could do that in this PR (and also do it in our repo), or postpone renaming to when we migrate full repo here.

I would do it in a future PR. We might also need to rename some traits and structs. It might require more attention.

**Review by bkchr — APPROVED** [review]

**Review by ggwpez — APPROVED** [review]

**Review by bkontur — APPROVED** [review]

**Review by sergejparity — APPROVED** [review]

**Review by EgorPopelyaev — APPROVED** [review]

**Pull request merged**